proto/osmosis/concentrated-liquidity/tx.proto

```diff
@@ -75,26 +75,12 @@ message MsgCreatePositionResponse {
 // ===================== MsgWithdrawPosition
 message MsgWithdrawPosition {
   uint64 position_id = 1 [ (gogoproto.moretags) = "yaml:\"position_id\"" ];
-  uint64 pool_id = 2 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
-  string sender = 3 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
-  int64 lower_tick = 4 [ (gogoproto.moretags) = "yaml:\"lower_tick\"" ];
-  int64 upper_tick = 5 [ (gogoproto.moretags) = "yaml:\"upper_tick\"" ];
-  string liquidity_amount = 6 [
+  string sender = 2 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
```

Comment:
Is there a reason why `sender` is after `position_id` here but before it in `MsgCollectFees` and `MsgCollectIncentives`?

Reply:
No reason, reordered here https://github.com/osmosis-labs/osmosis/pull/4695/commits/e3134ef4cdf4d416d4af39cae33099237b02eaae